db/migrate/20200123163055_create_tag_sets.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTagSets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :tag_sets do |t|
+      t.string :name
+      t.string :uuid
+      t.timestamps
+    end
+  end
```

Comment:
I only added one for Tags because I tried to reverse it locally and rails complained. It didn't complain about Tag Sets so think I can let the default do its thing?